linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -163,10 +162,13 @@ class UsageDataTelemeter(
   log.info(s"connecting to usageData proxy at $metricsDst")
   val client = Client(metricsService, config, pid, orgId)
 
-  val adminHandlers = Seq(
-    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry))
+  override def adminHandlers: Seq[Handler] = Seq(
+    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry)),
+    Handler("/usage", new UsageDataAdminHandler)
```

Comment:
right, i guess my question going forward is: should each project be responsible for serving its own static assets? today admin is the only project doing this.